configs/_base_/models/parta2.py

```diff
@@ -144,8 +144,8 @@
         rpn_proposal=dict(
             nms_pre=9000,
             nms_post=512,
-            max_num=512,
-            nms_thr=0.8,
+            max_per_im=512,
```

Comment:
For the parta2-based model, we use the anchor3d_head instead of anchor_head in mmdetection. The interface of the anchor3d_head keeps unchanged. More details can be found [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/dense_heads/anchor3d_head.py#L507) and [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/core/post_processing/box3d_nms.py#L68).